samples/java_springboot/19.custom-dialogs/src/main/java/com/microsoft/bot/sample/customdialogs/SlotFillingDialog.java

```diff
@@ -31,11 +31,11 @@
     // However, rather than persisting an index we will persist the last property we prompted for.
     // This way when we resume this code following a prompt we will have remembered what property
     // we were filling.
-    private static final String SLOT_NAME = "slot";
-    private static final String PERSISTED_VALUES = "values";
+    private final String SLOT_NAME = "slot";
+    private final String PERSISTED_VALUES = "values";
```

Comment:
As the props are not static, we should change the nomenclature here
```suggestion
    private final String slotName = "slot";
    private final String persistedValues = "values";
```